.travis.yml

```
@@ -1,58 +1,28 @@
 language: python
+python:
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.7
+  - 3.8
+  - 3.9-dev
+  - pypy
+  - pypy3
 matrix:
```

```suggestion
jobs:
```
Matrix is deprecated in favor of Jobs.  See Travis CI build config validation